blurb/blurb.py

```diff
@@ -956,9 +956,14 @@ def init_tmp_with_template():
             failure = str(e)
 
         if not failure:
-            assert len(blurb) # if parse_blurb succeeds, we should always have a body
+            assert len(blurb)  # if parse_blurb succeeds, we should always have a body
             if len(blurb) > 1:
                 failure = "Too many entries!  Don't specify '..' on a line by itself."
+            gh_issue_number = blurb[0][0]["gh-issue"]
+            if not gh_issue_number.isdigit():
+                failure = "The gh-issue number must be a number."
+            if int(gh_issue_number) < 32426:
+                failure = "The gh-issue number should be 32426 or above."
```

Comment:
Bit of whitespace for readability, and wording:
```suggestion
            gh_issue_number = blurb[0][0]["gh-issue"]

            if not gh_issue_number.isdigit():
                failure = "The gh-issue number must be a number."

            if int(gh_issue_number) < 32426:
                failure = "The gh-issue number must be 32426 or above."
```

Reply:
Removed completely from here since we should be raising an exception instead of displaying the failure.